app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    if not friends:
+        return "No friends to check."
+
+    vaccinated = True
+    no_mask_count = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotVaccinatedError:
+            vaccinated = False
```

Comment:
Think how you can structure your exceptions so you fulfil this requirement and you don't need to use `vaccinated` variable:

> If at least one of the friend has problems with vaccines the function should return a message `"All friends should be vaccinated".